d2render/d2ui/checkbox.go

```diff
@@ -46,7 +46,7 @@ func (v Checkbox) Draw(target *ebiten.Image) {
 		Filter:        ebiten.FilterNearest,
 	}
 	opts.GeoM.Translate(float64(v.x), float64(v.y))
-	if v.pressed == false {
+	if v.checkState == false {
```

Comment:
I know this is merged already, but a suggestion for booleans:

```diff
-if v.checkState == false {
+if !v.checkState {
```

The same goes for `v == true`